src/index.html

```diff
@@ -17,16 +18,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="form-search"
+    >
+      <div class="form-search__image form-search__image_big"></div>
```

Comment:
The task requires using the `background-image` CSS property for the icon instead of an `img` tag. Ensure that the icon is implemented using CSS as specified in the task requirements.